deploy/cloudformation/manifest-pipeline-pipeline.yml

```diff
@@ -322,34 +352,30 @@ Resources:
               aws:SecureTransport: false
```

Comment:
Is SecureTransport being set to `false` due to certificate issues, and if so, have we captured that this should be fixed once that concern has been resolved?

Reply:
This is on the pipeline artifacts bucket to deny insecure traffic to the bucket. This is probably not required here, but I can't see it hurting to have it.